packages/theming/apple-theme/src/appleColors.macos.ts

```diff
@@ -58,7 +59,7 @@ export function getAppleSemanticPalette(): AppleSemanticPalette {
 function getFluentUIApplePalette(): FluentUIApplePalette {
   const appearance = Appearance.getColorScheme();
   const mode = getCurrentAppearance(appearance, 'light');
-  const macOSAliasColorTokens = createMacOSAliasTokens(mode);
+  const macOSAliasColorTokens = createMacOSAliasTokens(mode, isHighContrastEnabled);
```

Comment:
Now that we have a getter function we can use it here, and not export isHighContrastEnabled directly.